toplevel/topdirs.ml

```diff
@@ -83,7 +90,7 @@ let _ = add_directive "directory" (Directive_string dir_directory)
 (* To remove a directory from the load path *)
 let dir_remove_directory s =
   let d = expand_directory Config.standard_library s in
-  Config.load_path := List.filter (fun d' -> d' <> d) !Config.load_path;
+  if true then assert false;
```

Comment:
What's the plan w.r.t. remove_directory?

Reply:
I believe the following will give the same behavior as before: we look at all the visible modules in the environment and remove all the ones that are persistent and not yet loaded.

Reply:
What do you mean?  The new code will always `assert false`, right?

Reply:
Yh, I just haven't implemented it yet

Reply:
Ah ok, I misunderstood your previous note.